mne/io/bti/bti.py

```diff
@@ -894,7 +894,8 @@ def _read_bti_header_pdf(pdf_fname):
 def _read_bti_header(pdf_fname, config_fname, sort_by_ch_name=True):
     """ Read bti PDF header
     """
-    info = _read_bti_header_pdf(pdf_fname)
+
+    info = _read_bti_header_pdf(pdf_fname) if pdf_fname is not None else dict()
```

Comment:
or I guess it must get cast later, never mind. Feel free to add a comment if you want but not 100% necessary